concepts/control-transfer/about.md

```diff
@@ -1,39 +1,34 @@
-# About
+# Introduction
 
-The normal control flow of loops in Swift can be altered using [Swift's control transfer keywords][control-transfer]. One of these keywords, `return` has been seen before in the basics concept exercise. With the use of `return`, if the loop is inside a function, the function will exit at that point, returning whatever value is specified, just as it would at any other point in a function. Two more control transfer keywords that are often used with loops are `continue` and `break`.
+Swift has a set of statements that can be used to alter the normal control flow of loops.
+These are known as control transfer statements.
+You have already seen `return`; this concept will introduce other, which include `continue`, `break`, and labels.
+We won't cover `fallthrough` in this concept, but you can read about it in the [Swift book][fallthrough].
```

Comment:
Is Fallthrough covered? I see it below.

Reply:
The introduction doesn't cover it to keep the text concise but you are right, that shouldn't be in the about docs.